python/lsst/daf/persistence/__init__.py

```diff
@@ -22,6 +22,10 @@
 
 """Python interface to lsst::daf::persistence classes
 """
+import pkgutil
+import lsstimport
+__path__ = pkgutil.extend_path(__path__, __name__)
+
```

Comment:
Why is this needed?  I'm concerned it may be fragile to have an `__init__.py` file that has both the above invocation and any real content (such as the imports below).  It implies that there is another `__init__.py` with the same Python name (i.e. `lsst.daf.persistence`) in another location, and I believe only one of those is ever executed.